src/styles/card/card.scss

```diff
@@ -0,0 +1,113 @@
+.main {
+  padding: 50px 40px;
+  height: 862px;
+  max-width: 1200px;
+
+  /* Центрування елемента горизонтально */
+  margin: 0 auto;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  grid-template-rows: repeat(2, 408px);
+  gap: 46px 48px;
+
+  justify-content: center;
+  align-items: center;
+}
+
+@media (min-width: 488px) {
+  .main {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .main {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .main {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  // width: 200px;
+  border: 1px solid white;
+  box-sizing: border-box;
+  background-color: #fff;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: transform 300ms;
+
+  &__title {
+    margin: 40px 0 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__sudtitle {
+    margin: 4px 0 0;
+    padding-top: 0;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+    font-size: 10px;
+  }
+
+  &__price-container {
+    margin-top: 24px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__price-label {
+    margin: 0;
+    color: #616070;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__money {
+    margin: 0;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__buy-button {
+    margin: 16px 0 0;
+    background-color: #00acdc;
+    position: relative;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    color: #fff;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-decoration: none;
+    box-sizing: border-box;
+    &:hover {
+      background: #fff;
+      color: #00acdc;
+      border: 1px solid #00acdc;
+    }
+  }
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The hover effect causes the card to overflow its container. Consider changing the transform property to `scale(1.02)` or `scale(1.05)` to prevent this issue.